### DISCUSSION TITLE
GSoC 2023: Enquiry

### DISCUSSION THREAD

**Discussion by Dhruv88** [PyLops/pylops#501] [Q&A] [answered]
Hello, This is Dhruv Deshmukh a final year student in computer science engineering at IIT Bhilai. I went through the project ideas list and found the **Project 2: MPI backend for distributed inverse problems** interesting. I tried to join the Slack channel but am unable to join since my account is registered in the workspace. It will be great if I can get an invite. I have experience in Python and also MPI. I would like to what will be a good place to start understanding and working on this project? @mrava87 @hongyx11 @cako

**Comment by mrava87** [comment, marked as answer]
Hello, please send me your email so I can add you to slack :)

**Comment by mrava87** [comment]
Oh I found it in your profile, you should have an invite
